dashboard/app/views/plc/tasks/edit/_common_editable.html.haml

```diff
@@ -1,6 +1,6 @@
 = f.label :name
 = f.text_field :name
-= f.label :learning_module_id
-= f.select :plc_learning_module_id, options_for_select(options_for_plc_task_learning_modules, @task.plc_learning_module_id)
+= f.label :learning_modules
+= f.select :plc_learning_module_ids, options_for_select(options_for_plc_task_learning_modules, @task.plc_learning_modules.pluck(:id)), {}, {multiple: true, size: 5}
```

Comment:
Make this larger and wider. It won't accommodate modules with long names. Also 5 is a small number and makes for a lot of scrolling. Maybe make size min(25, # modules) or something?

Alternatively, use selectize. But we don't *need* that